app/components/Sharing/index.tsx

```diff
@@ -11,9 +11,12 @@ import { CharacterType } from '@/lib/types';
 const BASE58 = '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz';
 const b58 = base(BASE58);
 
-/** Given a UUID string, returns the short key to access it via the sharing page. */
-export function uuidToShareKey(uuid: string) {
-  const uuidObj = new Uuid(uuid);
+/** Given a room name, returns the short key to access it via the sharing page. */
+export function roomNameToShareKey(roomName: string) {
+  if (roomName.indexOf('Fixie_') !== 0) {
```

Comment:
suggest treating it as a bare UUID if Fixie_ isn't present, I think we'll do that next on the server